packages/o2-blocks/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/o2-blocks",
-	"version": "1.0.0",
+	"version": "1.0.0-alpha.0",
```

Comment:
Do we want alpha status here? We don't intend to publish this for the foreseeable future I think :thinking:

Reply:
Doesn't matter much. Just go 1.0.0, as you note it's private and requires another change to be publishable.